include/zet_ddi.h

```diff
@@ -565,9 +565,9 @@ typedef ze_result_t (ZE_APICALL *zet_pfnMetricCreateFromProgrammableExp_t)(
 /// @brief Table of MetricExp functions pointers
 typedef struct _zet_metric_exp_dditable_t
 {
+    zet_pfnMetricCreateFromProgrammableExp_t                    pfnCreateFromProgrammableExp;
     zet_pfnMetricDestroyExp_t                                   pfnDestroyExp;
     zet_pfnMetricCreateFromProgrammableExp2_t                   pfnCreateFromProgrammableExp2;
-    zet_pfnMetricCreateFromProgrammableExp_t                    pfnCreateFromProgrammableExp;
```

Comment:
The spec shows this API as being added in 1.11.   The change was made here  https://github.com/oneapi-src/level-zero-spec/commit/bd119b848b666de6494053e61c425d249e751746

This is the reason it is at end of table.

Reply:
that should not have changed, the old api should have remained 1.9, that is the bug.